opencti-platform/opencti-front/src/private/components/data/csvMapper/CsvMapperForm.tsx

```diff
@@ -213,11 +224,19 @@ const CsvMapperForm: FunctionComponent<CsvMapperFormProps> = ({
                   <Typography>{t('Semicolon')}</Typography>
                 </div>
               </div>
-              <div className={classes.center} style={{ marginTop: 20 }}>
+              <div className={`${classes.center} ${classes.marginTop}`}>
+                  <Field
+                    component={TextField}
+                    name="skipLineChar"
+                    value={values.skipLineChar}
+                    label={t('Char to escape line')}
+                    onChange={(event: SelectChangeEvent) => setFieldValue('skipLineChar', event.target.value)}
```

Comment:
You are not using the proper TextField :(

Reply:
@Kedae Sorry, I use the same as the Name field. I can rewrite it.
I check on the project. 
Should I use the material one or the custom TextField from project ? 
`/src/components/TextField.jsx`

Reply:
I use the custom one. Fixed.